### DISCUSSION TITLE
How do I calculate the time, in minutes, from a log?

### DISCUSSION THREAD

**Discussion by scottjmaddox** [KellerJordan/modded-nanogpt#53] [Q&A] [answered]
I'm trying to make sure I'm comparing to the correct number. Based on the reported record of 3.80 minutes, I thought [these runs I did](https://github.com/KellerJordan/modded-nanogpt/pull/47#issuecomment-2537661051) were on a slower 8xH100 node. But the final values for `train_time` that I extracted from the [logs for the latest record](https://github.com/KellerJordan/modded-nanogpt/tree/master/records/121024_MFUTweaks) correspond to an average of 3.85 minutes (see below). I must be looking at the wrong number, or something. Can anyone point me in the right direction? Thanks!

Final train_time (ms) | Final train_time (min)
-- | --
231445 | 3.86
230850 | 3.85
230630 | 3.84
231289 | 3.85
230781 | 3.85
230778 | 3.85
230787 | 3.85
231439 | 3.86
231049 | 3.85
231219 | 3.85
231133 | 3.85
229395 | 3.82
231131 | 3.85
231003 | 3.85
231721 | 3.86
230409 | 3.84
231460 | 3.86
231056 | 3.85
230924 | 3.85
230621 | 3.84
231448 | 3.86
231093 | 3.85
231574 | 3.86
231402 | 3.86
231244 | 3.85
231133 | 3.85
230776 | 3.85
230775 | 3.85
231263 | 3.85
230962 | 3.85
231214 | 3.85
231170 | 3.85
230009 | 3.83
230925 | 3.85
231143 | 3.85
229966 | 3.83
231478 | 3.86
230970 | 3.85
231133 | 3.85
231777 | 3.86
  |  
Average: | 3.85

**Comment by YouJiacheng** [comment, marked as answer]
Keller run these logs on a slower machine too. I ran the master and got ~227.5s.

**Reply by scottjmaddox** [reply]
Ahh, gotcha, so the logs in the record folder are Keller's runs, not your runs. I assumed he had copied them from your tgz, but I see now that they're different. That explains it.

**Reply by KellerJordan** [reply]
yeah, I should actually be using YouJiacheng's runs to be honest.
